### PR TITLE
Fix triggers e2e failure by replacing pipeline resources by task

### PR DESCRIPTION
**Pull request by savitaashture**
# Changes
Fixes : https://github.com/tektoncd/triggers/issues/1515

Issue: Triggers integration tests are failing for below reason
```
Error from server (BadRequest): error when creating "/home/prow/go/src/github.com/tektoncd/triggers/docs/getting-started/pipeline.yaml": admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "resources"
Error from server (BadRequest): error when creating "/home/prow/go/src/github.com/tektoncd/triggers/docs/getting-started/pipeline.yaml": admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "resources"
Error from server (BadRequest): error when creating "/home/prow/go/src/github.com/tektoncd/triggers/docs/getting-started/pipeline.yaml": admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "resources"
```
Because latest Pipeline release has removed support of `Resources`

Changes: Removed all resources from getting-started example and replaced by Task resolver

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
Removed Pipeline resources from Triggers
```

### CODE REVIEW

**Comment by khrm** [comment]
/test pull-tekton-triggers-integration-tests
--


<br class="Apple-interchange-newline">

**Comment by khrm** [comment]
Please merge this https://github.com/savitaashture/triggers/pull/1 to your branch. It should fix this.

**Comment by khrm** [comment]
I would create a separate pr for other resources later on but we are in a chicken-and-egg situation. We need both fixes to get CI running.

**Comment by savitaashture** [comment]
@khrm Now CI failing because of this reason
```
    eventlistener_test.go:383: EventListener is ready
    eventlistener_test.go:518: Failed to create ResourceTemplate pr1: timed out waiting for the condition
```

We have `PipelineResource` here https://github.com/tektoncd/triggers/blob/main/test/eventlistener_test.go#L135-L149 in `TestEventListenerCreate` e2e 

Need to update the E2E by removing PipelineResource

**Comment by khrm** [comment]
I am changing it to Pipelineruns.

File: pkg/apis/triggers/v1alpha1/event_listener_validation.go [hunk 1]
```diff
@@ -37,6 +39,13 @@ var (
 	)
 )
 
+var _ resourcesemantics.VerbLimited = (*EventListener)(nil)
```
Comment (dibyom) [inline pkg/apis/triggers/v1alpha1/event_listener_validation.go:42]: why do we need this in this PR?

Comment (dibyom) [reply pkg/apis/triggers/v1alpha1/event_listener_validation.go:42]: (to delete old v1alpha1 ELs?)

**Review by dibyom — APPROVED** [review]

**Pull request merged**